src/components/outline-nav/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { OutlineListItems } from './components'
+import type { OutlineNavProps } from './types'
+import s from './outline-nav.module.css'
+import { useId } from '@react-aria/utils'
```

Comment:
TIL about this hook. 👏🏻 Also could we move this import to the top ⛏️

Reply:
✅ moved to the top!